src/nerve_tools/configure_nerve.py

```diff
@@ -142,11 +143,22 @@ def generate_subconfiguration(
                         'headers': healthcheck_headers,
                     },
                 ],
-                'labels': labels,
+                'labels': default_labels,
             }
 
-            v2_key = '%s.%s:%d.v2.new' % (
-                service_name, zk_location, port,
+            if typ != discover:
+                continue
```

Comment:
Can you add a comment explaining why we move on if we are here?

Is it because we only do the extra_advertise generated labels on the location type that fits what the user has asked to discover against?

Does that mean on there will be no v2_key when they don't match up?

In other words, can you confirm this is supposed to be a continue, and not and if/then?